.github/workflows/codecov.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       DJANGO_ENV: 'TESTING'
     strategy:
       matrix:
-        python: ["3.7"]
+        python: ["3.10"]
```

Comment:
Is there a reason to not use python 3.11 here?

Reply:
No particular reason, I'll change it to 3.11 (as long as Django supports it which is the case).

It should be the same version as used on the server. The Docker image in `Dockerfile` is used on the server. So I'll update it there as well.

Reply:
Actually that Python version is changed in #224. I'll wait till that one is merged @DutcherNL :smile: